homeassistant/components/mqtt/mixins.py

```diff
@@ -597,7 +616,9 @@ async def async_will_remove_from_hass(self):
     @property
     def available(self) -> bool:
         """Return if the device is available."""
-        if not self.hass.data[DATA_MQTT].connected and not self.hass.is_stopping:
+        mqtt_data: MqttData = self.hass.data[DATA_MQTT]
+        client = cast(MQTT, mqtt_data.client)
```

Comment:
```suggestion
        client: MQTT = self.hass.data[DATA_MQTT].client
```
I believe you used that shortcut before. You could also use `assert` here, too.